tests/e2e/src/pages/main.page.ts

```diff
@@ -240,6 +249,11 @@ export class MainPage extends BasePage {
     await this.click(this.menuElement + selector);
   }
 
+  async clickExplorerElement(text: string) {
+    const selector = `//*[text()='${text}']`;
```

Comment:
probably it is very common selector and we can put it on the top to re-call it next time.

Reply:
Not relevant anymore after the Allure fix